packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotHeader/ChatbotHeaderDrawer.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { ChatbotDisplayMode } from '@patternfly/virtual-assistant/dist/dynamic/Chatbot';
+import ChatbotConversationHistoryNav from '@patternfly/virtual-assistant/dist/dynamic/ChatbotConversationHistoryNav';
+import {
+  ChatbotHeader,
+  ChatbotHeaderMain,
+  ChatbotHeaderMenu
+} from '@patternfly/virtual-assistant/dist/dynamic/ChatbotHeader';
+interface navItem {
+  [key: string]: { id: string; text: string }[];
+}
+
+interface Conversation {
```

Comment:
I couldn't work out a good way to import this from the parent - is there a place I can stick types? It isn't being picked up as an export.

Reply:
I think if you want to export it, you'll need to move it into the src directory. if it's only needed for an example, it should be available to every object in the file as a private object, no?

Reply:
I'm using it in the component code as well as a couple of examples right now. It might be helpful for consumers to have the type.